src/editors/playgroundController.ts

```diff
@@ -168,9 +168,9 @@ export default class PlaygroundController {
       codeToEvaluate
     );
 
-    if (result) {
+    if (result && this._telemetryController?.needTelemetry()) {
```

Comment:
Same here - I think because we check if (this.needTelemetry()) { in the telemetry controller we can remove this if?

Reply:
I would keep this one as well. This parsing is not very heavy, but anyway we don't want extra time consumption, right?